docs/contributing/how-to-open-a-pull-request.md

```diff
@@ -1,135 +1,135 @@
+- Você pode verificar o nome e a URL remotos a qualquer momento:
   ```shell
   git remote -v
   ```
-- Fetch the latest changes from Gatsby:
+- Obtenha as alterações mais recentes do Gatsby:
   ```shell
   git fetch upstream master
   ```
-- [In the branch](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging) you want to update, merge any changes from Gatsby into your fork:
+- [Na branch](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging) que você deseja atualizar, faça merge de todas as alterações do Gatsby no seu fork:
   ```shell
   git merge upstream/master
   ```
-  - If there are any [merge conflicts](https://help.github.com/en/articles/resolving-a-merge-conflict-on-github), you'll want to address those to get a clean merge.
-- Once your branch is in good working order, push the changes to your fork:
+  - Se houver alguns [conflitos de mesclagem](https://help.github.com/pt/articles/resolving-a-merge-conflict-on-github), você vai querer resolver-los para obter um merge limpo.
+- Quando sua branch estiver funcionando em boas condições, faça push das alterações para o seu fork:
   ```shell
   git push origin head
   ```
 
-For more information on working with upstream repos, [visit the GitHub docs](https://help.github.com/en/articles/configuring-a-remote-for-a-fork).
+Para mais informações sobre como trabalhar com repositórios upstream, [visite a documentação do GitHub](https://help.github.com/pt/articles/configuring-a-remote-for-a-fork).
 
-_**Note:** as a member of the Gatsby repo, you can also clone it directly (instead of forking and using an upstream remote workflow). You can then push changes to [feature branches](https://git-scm.com/book/en/v1/Git-Branching-Branching-Workflows) to open PRs._
+_**Nota:** como membro do repositório Gatsby, você também pode cloná-lo diretamente (em vez de fazer um fork e usar um fluxo de trabalho remoto upstream). Você pode então enviar via push as alterações para [feature branches](https://git-scm.com/book/en/v1/Git-Branching-Branching-Workflows) para abrir PRs._
 
-## Additional resources
+## Recursos adicionais
 
 - CSS Tricks: [How to Contribute to an Open Source Project](https://css-tricks.com/how-to-contribute-to-an-open-source-project/)
-- [Creating a pull request](https://help.github.com/en/articles/creating-a-pull-request) from GitHub
-- [Configuring a remote for a fork](https://help.github.com/en/articles/configuring-a-remote-for-a-fork)
-- [Which remote URL should I use?](https://help.github.com/en/articles/which-remote-url-should-i-use)
-- [Git Branching and Merging](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging)
+- [Criar uma pull request](https://help.github.com/pt/articles/creating-a-pull-request) do GitHub
+- [Configurar remote para bifurcação](https://help.github.com/pt/articles/configuring-a-remote-for-a-fork)
```

Comment:
"bifurcação" => "fork"

Você está utilizando fork em todo o restante do doc.

Reply:
Por ser uma referência, eu trouxe o título do artigo:
![image](https://user-images.githubusercontent.com/5200384/68024272-657db600-fc88-11e9-9d5f-471826802dd6.png)
https://help.github.com/pt/github/collaborating-with-issues-and-pull-requests/configuring-a-remote-for-a-fork

Reply:
Done: https://github.com/gatsbyjs/gatsby-pt-BR/pull/29/commits/af1ac2d31b6cca5b70b0a1beeaba1b9ae79b8735

Reply:
Ah beleza! sem problemas então.